backend/core/package.json

```diff
@@ -97,7 +97,10 @@
     "typeorm": "0.2.41",
     "typeorm-naming-strategies": "^1.1.0",
     "typescript": "4.6.4",
-    "uuid": "^8.3.2"
+    "uuid": "^8.3.2",
+
```

Comment:
nit: should there be a space here?

Reply:
That was an artifact of testing which dev packages were actually needed for migration.  I took it out